src/pal/inc/rt/palrt.h

```diff
@@ -1502,6 +1502,7 @@ typedef struct _DISPATCHER_CONTEXT {
     PRUNTIME_FUNCTION FunctionEntry;
     DWORD EstablisherFrame;
     DWORD TargetIp;
+    PCONTEXT CurrentRecord;
```

Comment:
@janvorli Does it work for nested exceptions? As I understand, Caller SP will points to the frame by ``CallEHFunclet`` when nested exceptions happen.